lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -41,13 +42,15 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
     - `content_type` - either audio or video
     - `header_extension` - extension of the header file (for example .mp4 for CMAF)
     - `segment_extension` - extension of the segment files (for example .m4s for CMAF)
+    - `segment_duration` - segment duration range that should appear in a playlist.
+    - `target_partial_segment_duration` - expected duration of each partial segment, nil if not partial segments are expected
     - `header_naming_fun` - a function that generates consequent header names for a given track
     - `segment_naming_fun` - a function that generates consequent segment names for a given track
-    - `target_segment_duration` - expected duration of each segment
-    - `target_partial_segment_duration` - expected duration of each partial segment, nil if not partial segments are expected
     - `target_window_duration` - track manifest duration is kept above that time, while the oldest segments
                 are removed whenever possible
     - `persist?` - determines whether the entire track contents should be available after the streaming finishes
+    - `mode`-   Tells if the session is live or a VOD type of broadcast. It can influence type of metadata
```

Comment:
I now that I've created that comment at first but it is not that great. VOD content can't really be called a broadcast.

How about:
`track's mode that dictates type of metadata inserted into playlist's manifest`